### PR TITLE
schedules flaky tests

### PR DESCRIPTION
**Pull request by acruzgon**
This PR is:

- Removing the usage of `Global Project`
- Dividing tests into more `describe` blocks. 
- [AAP-26621](https://issues.redhat.com/browse/AAP-26621) Skipping `can create a simple schedule of resource type Workflow job template, then delete the schedule` because depends on how many workflow templates are at the moment of running the test

### CODE REVIEW

**Comment by nixocio** [comment]
@acruzgon there are schedule tests failing as part of this run, the credential ones I am adding another PR to take care of them.

File: cypress/e2e/awx/views/schedules.cy.ts [hunk 1]
```diff
@@ -8,52 +8,55 @@ import { Schedule } from '../../../../frontend/awx/interfaces/Schedule';
...
+    it('can create a simple schedule or resource type Job template, navigate to schedule details, then delete the schedule from the details page', () => {
       cy.navigateTo('awx', 'schedules');
       cy.verifyPageTitle('Schedules');
+      const scheduleName = 'E2E Simple Schedule' + randomString(4);
       cy.getBy('[data-cy="create-schedule"]').click();
       cy.selectDropdownOptionByResourceName('schedule_type', 'Job template');
       cy.selectDropdownOptionByResourceName('job-template-select', `${jobTemplate.name}`);
-      cy.get('[data-cy="name"]').type(`${scheduleName}`);
+      cy.getByDataCy('name').type(`${scheduleName}`);
       cy.selectSingleSelectOption('[data-cy="timezone"]', 'Zulu');
```
Comment (nixocio) [inline cypress/e2e/awx/views/schedules.cy.ts:54]: hey @acruzgon could we try to replace the`selectSingleSelectOption` with the `singleSelectByDataCy`?

**Review by AlexSCorey — APPROVED** [review]
My issues aren't blockers so I'm approving. However, I do recommend avoiding creating/syncing a project with each test, if we can.

File: cypress/e2e/awx/views/schedules.cy.ts [hunk 2]
```diff
@@ -73,27 +76,150 @@ describe.skip('Schedules - Create and Delete', function () {
...
+    let project: Project;
+
+    beforeEach(() => {
+      cy.createAwxOrganization().then((org) => {
+        organization = org;
+        cy.createAwxProject({ organization: organization.id }).then((proj) => {
+          project = proj;
+        });
+      });
+    });
```
[the remark this answers is not in the document]
Comment (acruzgon) [reply cypress/e2e/awx/views/schedules.cy.ts:90]: @AlexSCorey there's only one test in that describe block

**Comment by acruzgon** [comment]
@AlexSCorey not a big fan of that either but for some reason trying to edit/create some schedules with `Global Project` returns 500 errors

**Pull request merged**